src/Core/Math/Ellipsoid.js

```diff
@@ -55,39 +53,30 @@ Ellipsoid.prototype.setSize = function setSize(size) {
     this._radiiSquared = new THREE.Vector3(size.x * size.x, size.y * size.y, size.z * size.z);
 };
 
-const k = new THREE.Vector3();
-Ellipsoid.prototype.cartographicToCartesian = function cartographicToCartesian(coordCarto) {
+Ellipsoid.prototype.cartographicToCartesian = function cartographicToCartesian(coordCarto, target = new THREE.Vector3()) {
     const n = coordCarto.geodesicNormal.clone();
 
-    k.multiplyVectors(this._radiiSquared, n);
+    target.multiplyVectors(this._radiiSquared, n);
 
-    var gamma = Math.sqrt(n.dot(k));
+    const gamma = Math.sqrt(n.dot(target));
 
-    k.divideScalar(gamma);
+    target.divideScalar(gamma);
 
     n.multiplyScalar(coordCarto.altitude());
 
-    // n.multiplyScalar(0.0);
-
-    return k.add(n);
+    return target.add(n);
 };
 
-/**
- * @typedef {Object} EllipsoidCoordinate
- * @property {number} latitude
- * @property {number} longitude
- * @property {number} h - height
- */
 /**
  * Convert cartesian coordinates to geographic according to the current ellipsoid of revolution.
- *
  * @param {Object} position - The coordinate to convert
  * @param {number} position.x
  * @param {number} position.y
  * @param {number} position.z
- * @returns {EllipsoidCoordinate} an object describing the coordinates on the reference ellipsoid, angles are in degree
+ * @param {Coordinate} target coordinate to copy result
```

Comment:
nits: you can indicate that the parameter is optional (see http://usejsdoc.org/tags-param.html#optional-parameters-and-default-values)